Frameworks/UIKit/UIColor.mm

```diff
@@ -81,6 +87,7 @@ - (NSUInteger)retainCount {
 @Status Interoperable
 */
 - (void)dealloc {
+    [super dealloc];
 }
 
```

Comment:
@msft-Jeyaram I'll keep this, since it came in from develop. If we want it removed, we can remove it on develop.